index.html

```diff
@@ -16,11 +25,11 @@
     {% if announceyear > nowyear or announceday >= nowday and announceyear >= nowyear %}
       <div class ="announcement">
         {{announcement.output}}
-      </div> <!-- /.announcement -->
+      </div>
     {% endif %}
   {% endfor %}
 
-</section> <!-- /.intro-header -->
+</section>  -->
```

Comment:
hmm, yeah we should discuss on at our next meeting, gonna ping @garrying so we all look at the announcement functionality.